src/ray/rpc/server_call.cc

```diff
@@ -0,0 +1,19 @@
+#include "ray/rpc/server_call.h"
+
+#include "ray/common/ray_config.h"
+
+namespace ray {
+namespace rpc {
+
+static std::unique_ptr<boost::asio::thread_pool> executor_;
```

Comment:
Maybe we should join this?

Reply:
I'm not sure, but I feel no need to join (also a little bit tricky to join it. we can do).

Right now, it only calls Finish which will send event to the cq. I think grpc stop should handle this.

But I can't be 100% confident in this one.

I think the issue for the join is that, when should we join. This is a shared global thread pool among servers. So it should be joined after all thread pool destructed. And also the last one to join (it's a global vars).